apps/src/templates/teacherDashboard/OwnedSectionsTable.jsx

```diff
@@ -55,7 +55,12 @@ export const courseLinkFormatter = function (course, {rowData}) {
           <a
             href={
               showV2TeacherDashboard()
-                ? teacherDashboardUrl(rowData.id, assignmentPaths[0])
+                ? assignmentPaths[0].includes('/s/')
+                  ? teacherDashboardUrl(
```

Comment:
Hmmm. So the `/s` is coming from some sort of backend object? I'm not sure we just want to have this random logic to re-write the backend string in this class. 

What if instead we add a router redirect from `/s/` to `/unit` similar to what we do for `/manage_students` to `/roster`?

Reply:
1. we are already doing this in the file
2. we are constructing the url right here.

Because of those, my comment is incorrect and we can just merge as is